### DISCUSSION TITLE
Update itemfulfillment

### DISCUSSION THREAD

**Discussion by christophe-fms** [netsuitephp/netsuite-php#130] [Q&A]
Hi,

I'm trying to update an existing SalesOrder (add tracking and weight to item), witch is diffferent from creating a new one. Do you have any examples of doing such an operation ?
Best regards

**Comment by jrebs** [comment]
A sales order is just another kind of record and so updating it is like updating most records. You'd use the `UpdateRequest` class with the appropriate sales order as your record. That being said, it sounds to me like you don't actually need to update the sales order but instead you need to create or modify an `ItemFulfillment` record. This is a little different from normal record creation.

https://github.com/netsuitephp/netsuite-php/blob/master/EXAMPLES.md#Add-an-item-fulfillment
